### DISCUSSION TITLE
Bug: Music playback doesn't stop while playing videos

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2200] [General]
I'm not sure if that's a bug or just unexpected behaviour – but if I don't stop the music playback, it's still playing while I open a video from the meeting program list. I know that this is an edge case (because you have to stop the music before starting the meeting), but still sounds like a not very hard-to-implement fix.

**Comment by mtdvlpr** [comment]
Hi @ux-git, I did implement this a while back, but I got feedback that some users have a silent video that they start before the meeting with a countdown. They want to show the video while also playing the background music.

**Reply by ux-git** [reply]
It makes sense 👍
